CHANGELOG.md

```diff
@@ -5,7 +5,7 @@ x.x.x Release notes (yyyy-MM-dd)
 
 ### Fixed
 * <How to hit and notice issue? what was the impact?> ([#????](https://github.com/realm/realm-js/issues/????), since v?.?.?)
-* None.
+* Realm initialized the filesystem when being imported instead of waiting for the first Realm to be opened. (Issue [#2218] (https://github.com/realm/realm-js/issues/2218), since 2.22.0).
```

Comment:
```suggestion
* Realm initialized the filesystem when being imported instead of waiting for the first Realm to be opened. ([#2218] (https://github.com/realm/realm-js/issues/2218), since v2.22.0).
```